### PR TITLE
CI-Volunteer Mission Page Backend

### PR DESCRIPTION
**Pull request by Charul7788**

### CODE REVIEW

File: CI-API/Controllers/VolunteerMissionController.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            return await VolunteerMissionService.RecommandedCoworker(Email, missionId, userId);
+        }
+
+
+
+
+
+
+
+
```
Comment (UrvishaNayee) [inline CI-API/Controllers/VolunteerMissionController.cs]: Remove extra space

File: CI-API.Core/ViewModel/VolunteerMissionViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        public long? userId { get; set; }
+        public string? commentText { get; set; }
+        public string? approvalStatus { get; set; }
+        public bool? IsFavMission { get; set; }
+        public List<DocDetail> docsPath { get; set; }  
+        public string? imgPath { get; set; }        
+        public List<Comment> comments { get; set; }
+    }
+
+    public class DocDetail
```
Comment (UrvishaNayee) [inline CI-API.Core/ViewModel/VolunteerMissionViewModel.cs]: Rename class name to DocumentDetail

File: CI-API.Data/Repository/VolunteerMissionRepository.cs [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+        #region GetMissionData
+        public async Task<JsonResult> AllMissiondata(long missionId, long userId)
+        {
+            try
+            {
+                var isMissionId =await Task.FromResult(_cIDbContext.Missions.Where(m => m.MissionId == missionId).FirstOrDefault()) ;
+
+                if (isMissionId != null)
+                {
+                    var data1 = await Task.FromResult(from mission in _cIDbContext.Missions
```
Comment (UrvishaNayee) [inline CI-API.Data/Repository/VolunteerMissionRepository.cs]: You already get mission record above use that here do not get again from db.
rename 'isMissionId ' to mission

File: CI-API.Data/Repository/VolunteerMissionRepository.cs [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+        }
+
+        #endregion
+
+        #region AddToFavourite
+        public async Task<JsonResult> AddToFavourite(int missionId, int userId)
+        {
+            try
+            {
+                var ismidOrNot =await Task.FromResult(_cIDbContext.Missions.Where(m => m.MissionId == missionId).FirstOrDefault());
```
Comment (UrvishaNayee) [inline CI-API.Data/Repository/VolunteerMissionRepository.cs]: rename ismidOrNot to mission

**Review by UrvishaNayee — APPROVED** [review]

**Review by Trusha-Tatvasoft — APPROVED** [review]

**Pull request merged**